modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/SearchBar/index.js

```diff
@@ -24,6 +24,18 @@ const SearchBar = ({setFilters}) => {
 		}));
 	};
 
+	function setSearch() {
```

Comment:
Hi @thaynam! To improve our pattern, let's try to use arrow functions (https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Functions/Arrow_functions)!

Also, should be good choose a better name for this function, because "setSearch" may means like set some value in "search" state.

```suggestion
	const setSearch = () => {
```